packages/core/src/App/Containers/Layout/header/dashboard-header.jsx

```diff
@@ -36,8 +53,14 @@ const LoggedOutHeader = () => {
         <header className='dashboard-header dashboard-header--logged-out'>
             <div className='dashboard-header__left'>
                 <div onClick={() => history.push(routes.dashboard)}>
-                    <DerivLogo style={{ marginRight: '0.684rem' }} />
-                    <DerivText />
+                    {isDesktop() ? (
+                        <React.Fragment>
+                            <DerivLogo style={{ marginRight: '0.684rem' }} />
```

Comment:
any reason why we dont use class here?

Reply:
No particular reason, is it always preferred in this setup rather than inline styling?

Reply:
I think we better add a classname to this and set the margin there, otherwise it will be hard to maintain these styles later on